pkg/operator/features/feature_accessor.go

```diff
@@ -0,0 +1,23 @@
+package features
```

Comment:
Can you share the usecase for which the feature_accessor or the controller is required to monitor techpreview features?

Reply:
As discussed, we'll move to setting the addon features from OLM Subscription object so the operator's environment variable can consume it.